src/common/popup/components/external-link.jsx

```diff
@@ -1,20 +1,30 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
 function ExternalLink(props, context) {
   const handler = (props.href && context.openext)
     ? () => context.openext(props.href)
     : () => true;
 
-  // eslint-disable-next-line jsx-a11y/anchor-has-content, jsx-a11y/no-static-element-interactions
-  return (<a {...props} target="_blank" rel="noopener noreferrer" onClick={handler} />);
+  /* eslint-disable jsx-a11y/anchor-has-content, jsx-a11y/no-static-element-interactions */
+  return (
+    <a
+      target="_blank"
+      rel="noopener noreferrer"
+      onClick={handler}
+      onKeyDown={handler}
+      {...props}
```

Comment:
maybe we can add `role="link"` here and remove the `no-static-element-interactions` ignore.

Also when adding `props.children` as required and explicitly setting that into the body of the `<a>` tag, we could probably remove the other eslint disable and make sure that this element always has a body of some sort.

Reply:
I've added another commit to avoid the ESLint-rule override and explicitly required `children` as a prop 👋

Reply:
An anchor has the default role of `link`, adding it should be unnecessary. What’s this link linking to? I see a `target="_blank"` but no `href` — if this is only triggering some JS, perhaps it should be a `<button>`

Reply:
Thanks for your input @planktonic. The handler is only there for Safari, which did not allow the use of regular anchors within extensions to open new browser tabs (see `openext` function that's passed down in the React context).

My thinking was: It's still semantically a link even if Safari makes you jump through hoops.

The commit I was referring to above actually adds the `href` and removes the ESLint override 😉